### PR TITLE
Incite Flavor Text and More

### PR DESCRIPTION
**Pull request by jwoolley**

### CODE REVIEW

**Comment by jwoolley** [comment]
I screwed up obstacles pretty badly. Let me push a fix.

File: src/index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const grammar = tracery.createGrammar(the_pros);
 
 grammar.addModifiers(tracery.baseEngModifiers); 
 
-for (let i = 0; i < 10; ++i) {
+const numExamples = process.argv[2] || 10;
```
Comment (mdiehr) [inline src/index.js]: cool idea!!

File: src/grammar.json [hunk 2]
```diff
@@ -581,7 +653,9 @@
       "storm",
       "still",
       "tron",
-      "erator"
+      "erator",
+      "-troll",
```
Comment (mdiehr) [inline src/grammar.json]: “I just threw it together this morning……I wasn’t even planning on playing this event…..I was in the neighborhood and figured why not……” Then he made a face like this:
:-9

File: src/grammar.json [hunk 3]
```diff
@@ -228,31 +239,64 @@
...
+        "panic", "glee", "rage", "insecurity", "self-consciousness", "abject terror", "cautious optimism", "boredom", "cowardice",
+        "lust", "near-fanatical devotion", "arrogance", "pluckiness", "indifference", "unwillingness", "begrudging admiration"
+    ],
+    "itemNoun": [
+        "weapon", "card", "wand", "spellbook", "staff", "orb", "tree", "plant", "crown", "jewel", "amulet", "egg", "banner", "boot",
+        "aegis", "hedron", "altar", "portal", "door", "cluestone", "flask", "bident", "scepter", "idol", "mask", "sword", "longbow",
+        "grimoire", "chalice", "blade", "gift", "signet", "monolith", "gemstone", "clue", "scroll", "lantern", "collar", "talisman"
+    ],
+    "irregularPluralNoun": [
+        "teeth", "feet", "pants", "geese", "fish", "children", "men", "women", "potatoes", "knives"
```
Comment (mdiehr) [inline src/grammar.json]: From nothing came geese.

**Comment by mdiehr** [comment]
ok, I'll wait for your changes

**Comment by jwoolley** [comment]
Okay, it should be unbroken now!

**Comment by mdiehr** [comment]
This PR has been deployed to the bot, so it'll start generating even more cool news

**Comment by jwoolley** [comment]
Excellent. I've verified that the cheapbots source page is generating sample headlines

`BEHIND THE FLAVOR TEXT: "The curtain of rationality evaporated from the champion, revealing the lust and geese beneath."`

**Pull request merged**